### ISSUE TITLE
Microsoft Access: After activating check box, screen reader does not announce state of it.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15124]
### Steps to reproduce:
Open 'Access' application.
Activate Screen reader 'NVDA' application.
Navigate to 'Graphic1' database from recent file in backstage and activate it.
Navigate to 'All access object' pane in left side.
Navigate to 'Issue Details' table under 'Forms' in left pane.
'Issue Details' form will be appear and navigate to that form.
Navigate to 'Fixed' checkbox using tab key.
Change state of checkbox using space key.
Observe the screen reader issue.

### Actual behavior:
After activating check box, screen reader does not announce state of it.
Screen reader is announcing space only.

### Expected behavior:
After activating check box, screen reader should announce checked/unchecked state of it. 

### NVDA logs, crash dumps and other attachments:
<img width="960" alt="MAS4 1 2 After activating check box, screen reader does not announce state of it" src="https://github.com/nvaccess/nvda/assets/93735775/c0bfd531-0618-486a-afbf-d36d6286d6f6">

https://github.com/nvaccess/nvda/assets/93735775/cf5e6be7-0934-4583-ae52-c472141dcbda

https://github.com/nvaccess/nvda/assets/93735775/41732505-0153-4ac4-97c0-1bbbed1d11e4

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce this issue, we need minimal sample.
The steps in this issue refer to a file that is not attached to this issue

>Navigate to 'Graphic1' database from recent file in backstage and activate it.

We also need you to fill out the issue template.
Please provide the name and version of other software in use when reproducing the issue.

Kind Regards,
NV Access Software Developers

**Comment by msftedad** [comment]
@seanbudd:  Please find below the updated details, 
'Graphic1' is a file made on 'Windows Access' App not supported so adding shared location link below:
https://drive.google.com/file/d/1CUZeDPPUjTIGAZ0jg5pYjiI-xy_fgE8k/view?usp=sharing

System configuration: NVDA installed/portable/running from source:
NVDA version: NVDA 2023 (2023.1.0.27913)
Windows version: OS: 22H2 (OS Build 22621.1848)
Name and version of other software in use when reproducing the issue: N/A
Other information about your system: N/A
Other questions
Does the issue still occur after restarting your computer?  Yes.
Have you tried any other versions of NVDA? If so, please report their behaviors.  No.
If NVDA add-ons are disabled, is your problem still occurring? N/A
Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu? N/A

**Comment by seanbudd** [comment]
> 'Windows Access' App
>Name and version of other software in use when reproducing the issue: N/A

Again, please confirm which version of Windows Access is being used here

**Comment by msftedad** [comment]
@seanbudd: Windows Access Version: Microsoft® Access® for Microsoft 365 MSO (Version 2307 Build 16.0.16626.20028) 64-bit is uesd.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by Adriani90** [comment]
This is still an issue in NVDA alpha-53423,57f2e9dc (2026.1.0.53423) 
with Microsoft access 365 MSO (Version 2510 Build 16.0.19328.20190) 64 Bit.

Narator works as expected in this case.

**Comment by msftedad** [comment]
@Adriani90, As verified, the issue is reproducible only with NVDA.
